src/txn/txn_rollback_to_stable.c

```diff
@@ -146,6 +147,31 @@ __rollback_row_add_update(WT_SESSION_IMPL *session, WT_PAGE *page, WT_ROW *rip,
     return (ret);
 }
 
+/*
+ * __rollback_check_if_txnid_committed --
+ *     Check if the transaction id is committed.
+ */
+static bool
+__rollback_check_if_txnid_committed(WT_SESSION_IMPL *session, uint64_t txnid)
```

Comment:
The value this function returns and the function name doesn't match. Change the function name from `committed` to `uncommitted`?

Reply:
Fixed the function name.